data-sources/ajax/config.yml

```diff
@@ -0,0 +1,27 @@
+id: ajax
+displayName: Browser Ajax Requests 
+description: |
+**Benefits of understanding AJAX requests**
```

Comment:
This multi-line description needs to be indented one level in order to be valid. Here's an example of the first line of the description, but all of it needs to be indented:

```
description: |
  **Benefits of understanding AJAX requests**
  ...
```